x-pack/legacy/plugins/siem/server/lib/detection_engine/notifications/get_signals_count.ts

```diff
@@ -21,7 +21,7 @@ interface GetSignalsCount {
   ruleAlertId: string;
   ruleId: string;
   index: string;
-  kibanaUrl: string | undefined;
+  kibanaSiemAppUrl: string | {} | undefined;
```

Comment:
This can't actually be an `object`, but rather is a byproduct of the typing of `meta` correct?

https://github.com/elastic/kibana/blob/ca3c5ab90dbb21d491ba70eb4c08fe96a9a679f1/x-pack/legacy/plugins/siem/server/lib/detection_engine/types.ts#L55

If this indeed can only ever be a `string`, might be best to keep it as such and narrow the type coming from `meta` in these two files? Thoughts?

https://github.com/elastic/kibana/blob/7e8273001406d2ecc8bb88b7e603bb56fd82937b/x-pack/legacy/plugins/siem/server/lib/detection_engine/notifications/rules_notification_alert_type.ts#L50
https://github.com/elastic/kibana/blob/b9c8291a32831929fffc79e67815fa42aa479cde/x-pack/legacy/plugins/siem/server/lib/detection_engine/signals/signal_rule_alert_type.ts#L223